src/lib/capacityEngine.js

```diff
@@ -40,12 +43,17 @@ function capacityEngine(store, productIds, time) {
           .slice(firstUsableTrancheIndex) // skip unusable
           .reduce((total, free) => total.add(free), Zero);
 
+        const basePrice = product.useFixedPrice
+          ? targetPrice
+          : calculateBasePrice(targetPrice, bumpedPrice, bumpedPriceUpdateTime, now);
+
         return {
           capacityUsedNXM: used.mul(NXM_PER_ALLOCATION_UNIT).add(capacityUsedNXM),
           capacityAvailableNXM: available.mul(NXM_PER_ALLOCATION_UNIT).add(capacityAvailableNXM),
+          minBasePrice: minBasePrice.eq(Zero) ? basePrice : bnMin(minBasePrice, basePrice),
         };
       },
-      { capacityAvailableNXM: Zero, capacityUsedNXM: Zero },
+      { capacityAvailableNXM: Zero, capacityUsedNXM: Zero, minBasePrice: Zero },
```

Comment:
optional: minBasePrice can be initialized with max uint and that would avoid the ternary 3 lines above

Reply:
Actually nevermind, it would create a ternary lower in the code when calculating `annualPrice`